tests/test_bootstrap.py

```diff
@@ -1143,23 +1143,30 @@ async def test_bootstrap_empty_integrations(
     await hass.async_block_till_done()
 
 
-@pytest.mark.parametrize("integration", ["mqtt_eventstream", "mqtt_statestream"])
-@pytest.mark.parametrize("load_registries", [False])
-async def test_bootstrap_dependencies(
-    hass: HomeAssistant,
-    caplog: pytest.LogCaptureFixture,
-    integration: str,
-) -> None:
-    """Test dependencies are set up correctly,."""
+@pytest.fixture(name="mock_mqtt_config_flow")
+def mock_mqtt_config_flow_fixture() -> Generator[None, None, None]:
+    """Mock MQTT config flow."""
 
-    # Prepare MQTT config entry
     @HANDLERS.register("mqtt")
     class MockConfigFlow:
         """Mock the MQTT config flow."""
 
         VERSION = 1
         MINOR_VERSION = 1
 
+    yield
+    HANDLERS.pop("mqtt")
```

Comment:
Why aren't we using our common test helper?

https://github.com/home-assistant/core/blob/573cd8e94a5e061651897673fcc3ad00dfe33d45/tests/common.py#L1420